dashboard/app/controllers/api_controller.rb

```diff
@@ -436,7 +436,7 @@ def user_progress
     if current_user
       if params[:user_id].present?
         user = User.find(params[:user_id])
-        return head :forbidden unless user.student_of?(current_user)
+        return head :forbidden unless user.student_of?(current_user) || user.id == current_user.id
```

Comment:
note to self: needed because `queryUserProgress` eventually calls this, and `user.id == current_user.id` is true when a teacher queries their own progress as part of this feature